component-server-parent/component-server/src/test/java/org/talend/sdk/component/server/front/ConfigurationTypeResourceImplTest.java

```diff
@@ -124,36 +121,6 @@ void migrateUnexpectedWithNPE() {
         assertEquals("Migration execution failed with: unexpected null", error.getDescription());
     }
 
-    @Test
-    void migrateWithEncrypted() {
-        final JsonBuilderFactory factory = JsonProvider.provider().createBuilderFactory(emptyMap());
-        final JsonObject json = factory
-                .createObjectBuilder()
-                .add("configuration.url", "vault:v1:hcccVPODe9oZpcr/sKam8GUrbacji8VkuDRGfuDt7bg7VA==")
-                .add("configuration.username", "username0")
-                .add("configuration.password", "vault:v1:hcccVPODe9oZpcr/sKam8GUrbacji8VkuDRGfuDt7bg7VA==")
-                .build();
-        final Map<String, String> config = base
-                .path("/configurationtype/migrate/amRiYy1jb21wb25lbnQjamRiYyNkYXRhc2V0I2pkYmM/-2")
-                .request(APPLICATION_JSON_TYPE)
-                .post(Entity.entity(new HashMap<String, String>() {
-
-                    {
-                        put("configuration.url", "vault:v1:hcccVPODe9oZpcr/sKam8GUrbacji8VkuDRGfuDt7bg7VA==");
-                        put("configuration.username", "username0");
-                        put("configuration.password", "vault:v1:hcccVPODe9oZpcr/sKam8GUrbacji8VkuDRGfuDt7bg7VA==");
-                        put("configuration.connection.password",
-                                "vault:v1:hcccVPODe9oZpcr/sKam8GUrbacji8VkuDRGfuDt7bg7VA==");
-                    }
-                }, APPLICATION_JSON_TYPE))
-                .readEntity(Map.class);
-        assertEquals("true", config.get("configuration.migrated"));
-        assertEquals("5", config.get("configuration.size"));
-        assertEquals("vault:v1:hcccVPODe9oZpcr/sKam8GUrbacji8VkuDRGfuDt7bg7VA==", config.get("configuration.url"));
-        assertEquals("username0", config.get("configuration.username"));
-        assertEquals("test", config.get("configuration.connection.password"));
```

Comment:
Here we can check that instead of test we still have vault value, like for configuration.url. It's hard to get the test wirhout knowing structure before and after